token/cli/src/main.rs

```diff
@@ -1409,22 +1342,10 @@ async fn command_revoke(
         return Err(format!("No delegate on account {}", account).into());
     }
 
-    let instructions = vec![revoke(
-        &program_id,
-        &account,
-        &owner,
-        &config.multisigner_pubkeys,
-    )?];
-    let tx_return = handle_tx(
-        &CliSignerInfo {
-            signers: bulk_signers,
-        },
-        config,
-        false,
-        0,
-        instructions,
-    )
-    .await?;
+    let token = token_client_from_config(config, &program_id, &Pubkey::default());
```

Comment:
nit: The final parameter of `&Pubkey::default()` absolutely works but is kinda hacky. I don't feel too strongly about it, but how about also pulling out the mint pubkey in the first branch higher up, and using `Pubkey::default()` in the offline signing case with a comment explaining why it's ok.  Or at the very least, a comment explaining why this parameter is ok

Reply:
this probably sounds stupid but the lack of comments on anything has made me wonder if we arent supposed to include comments as a style thing

Reply:
I don't think there's any style in particular.  At the very least, I like to require comments where the code may not be clear, and beyond that, it's up to the person. In the monorepo, there are loads of PRs just adding comments to unclear code, so it's certainly appreciated.

Reply:
It'd actually be really cool to add robust rust-docs to spl-token-client (like we've done for solana-client), since that's a dependency that's likely to be pulled in by external rust users.